test/conformance/behaviors/sig-scheduling/preemption.yaml

```diff
@@ -0,0 +1,15 @@
+suite: scheduling/framework
```

Comment:
I feel like it'd be better if suite name matched path, or does this imply a single suite can be split up across multiple files?

Reply:
I'd prefer name matching, of specifying the schema in a more direct way.

Reply:
Some of the files may contain only one behavior, which is the main reason I grouped them. I *feel* like it makes sense here since the files in scheduling aren't tied to specific API resources, but am open for feedback.